### DISCUSSION TITLE
How to save sessions to DB?

### DISCUSSION THREAD

**Discussion by ggukggukee** [gladly-team/next-firebase-auth#712] [Q&A]
How I should save session to my DB? 
Should I get refresh token when user login and save to the db and delete that session when logout?

**Comment by kmjennison** [comment]
You don't need to save sessions yourself. This library relies on Firebase auth state that Firebase manages.

**Reply by ggukggukee** [reply]
but can i see list of all my sessions and manage them?

**Reply by kmjennison** [reply]
You can see them in the [Firebase console](https://console.firebase.google.com) and manage them there. Or, see [docs](https://firebase.google.com/docs/auth/admin/manage-sessions) for info on managing sessions programatically.
